go/vt/vttablet/tabletserver/gc/tablegc.go

```diff
@@ -166,7 +180,7 @@ func (collector *TableGC) Open() (err error) {
 		// already open
 		return nil
 	}
-	collector.lifecycleStates, err = schema.ParseGCLifecycle(*gcLifecycle)
+	collector.lifecycleStates, err = schema.ParseGCLifecycle(gcLifecycle)
 	if err != nil {
 		return fmt.Errorf("Error parsing -table_gc_lifecycle flag: %+v", err)
```

Comment:
```suggestion
		return fmt.Errorf("Error parsing --table_gc_lifecycle flag: %+v", err)
```

Reply:
(please check for other instances of this, which i meant to clean up previously .. sorry about that!)

Reply:
done